### PR TITLE
Add SX-Window 3.1 alternate dump

### PR DESCRIPTION
**Pull request by briceonk**
I found a boxed SX-Window 3.1 set that had two extra disks not in the already existing dumps. These images were dumped directly from the pictured floppy disks using a Greaseweazle and a Mitsumi 5.25" drive.

Images: https://archive.org/details/sxwindow-31.tar
IMG_4309

I've never updated a software list before, so let me know if I made any mistakes :)

### CODE REVIEW

File: hash/x68k_flop.xml [hunk 1]
```diff
@@ -16387,6 +16387,54 @@ Typing "CD PACMAN" and enter key, Typing "PAC" (or "PAC.BAT") and enter key. The
 		</part>
 	</software>
 
+	<software name="sxwin31b" cloneof="sxwin31">
+		<description>SX-Window v3.1 (Alt 2)</description>
```
Comment (angelosa) [inline hash/x68k_flop.xml:16391]: `alt `should be lowercase

Comment (briceonk) [reply hash/x68k_flop.xml:16391]: Done. Should I change the other alt to `SX-Window v3.1 (alt)` to keep it consistent?

Comment (angelosa) [reply hash/x68k_flop.xml:16391]: If there is one then yes, thanks.

Comment (briceonk) [reply hash/x68k_flop.xml:16391]: Done!

**Comment by happppp** [comment]
See here on how to contribute disk images: https://wiki.mamedev.org/index.php?title=Submitting_Source_Code
Freely distributable software is of course ok to link to here on github, but I presume these aren't.

**Comment by briceonk** [comment]
> See here on how to contribute disk images: https://wiki.mamedev.org/index.php?title=Submitting_Source_Code
> Freely distributable software is of course ok to link to here on github, but I presume these aren't.

Sure, I'll send it there. Sharp released SX-Window themselves years ago, so I didn't think it would be an issue. Sorry about that.

**Pull request merged**